### PR TITLE
Cloud teams

### PR DESCRIPTION
**Pull request by guimachiavelli**

### CODE REVIEW

File: learn/cloud/teams.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Meilisearch Cloud teams — Meilisearch documentation
+description: Meilisearch Cloud teams helps collaboration between project stakeholders with different skillsets and responsibilities.
+---
+
+# Meilisearch Cloud teams
+
+Meilisearch Cloud teams are groups of users who all have access a specific set of projects. This feature is designed to help collaboration between project stakeholders with different skillsets and responsibilities.
```
Comment (guimachiavelli) [inline learn/cloud/teams.mdx:8]:
```suggestion
Meilisearch Cloud teams are groups of users who all have access a to specific set of projects. This feature is designed to help collaboration between project stakeholders with different skillsets and responsibilities.
```

**Review by CaroFG — DISMISSED** [review]
lgtm! Not sure if it's worth mentioning, but a team owner can also rename the team

**Review by macraig — DISMISSED** [review]
+1 to @CaroFG 's comment, we can add that it's possible to edit the team name

**Comment by guimachiavelli** [comment]
Good suggestion, just updated the text so it mentions that. Everything else should be exactly the same.

**Review by macraig — APPROVED** [review]

**Pull request merged**